src/Nordea/Components/FileUpload.elm

```diff
@@ -209,7 +209,7 @@ view (FileUpload config) =
         inputFile =
             let
                 isDisabled =
-                    config.allowMultiple || List.isNotEmpty config.files
+                    not config.allowMultiple || List.isNotEmpty config.files
```

Comment:
Given how it was before, I think we must change it to && for it to be the same?

```suggestion
                    not config.allowMultiple && List.isNotEmpty config.files
```

Reply:
Yes, updated now 🫣